premailer/merge_style.py

```diff
@@ -31,7 +31,8 @@ def merge_styles(inline_style, new_styles, classes):
             inline_style(str): the old inline style of the element if there
             is one new_styles: a list of new styles, each element should be
             a list of tuple classes: a list of classes which maps
-            new_styles, important!
+            new_styles, important! unset_removes_attribute: allow us to remove
+            certain CSS attributes with rules that set their value to 'unset'
```

Comment:
It was already a strange sentence. But Did you perhaps mean to prefix the `unset_removes_attribute: allow us to ...` with some newlines?